src/kendo.numerictextbox.js

```diff
@@ -308,6 +310,16 @@ var __meta__ = { // jshint ignore:line
             that._downArrowEventHandler = new kendo.UserEvents(that._downArrow, { release: _release });
         },
 
+        _validation: function () {
+            var that = this,
+                element = that.element;
+
+            that._validationIcon = $("<span>")
+                .addClass("k-icon k-i-warning")

```

Comment:
Just and idea - you can extract the names of the classes into separate variables (constants), so they can be reused. It is also preferable to use the self-closing version of the tag, i.e. `$("<span />")`.

An alternative is to use something like:
```
var VALIDATION_ICON = "<span class='k-icon k-i-warning></span>";

that._validationIcon = $(VALIDATION_ICON).insertAfter()...
```
but this depends whether or not `k-icon` and `k-i-warning` are used in more than one places (the same applies to the icon rendering).